test/features/caseload_users_test.exs

```diff
@@ -141,4 +141,15 @@ defmodule Healthlocker.CaseloadUsersTest do
 
     assert session |> has_text?("Jimmy Smits")
   end
+
+  test "view tracking overview", %{session: session} do
```

Comment:
FYI: @Cleop could be an idea to check out the E2E tests in Healthlocker as a ref for LDMW. 👍